samples/ubuntu-16.04_py-3.5.1/conda-env.json

```diff
@@ -0,0 +1,11 @@
+{
+    "dependencies": [
+        "python=3.5.1",
+        "numpy",
+        {
+            "pip": [
+                "pandas"
+            ]
+        }
+    ]
+}
```

Comment:
This JSON was created with `src.utils.save_json()`. I updated that function to add a newline at the end of the file.